routes/users.js

```diff
@@ -0,0 +1,112 @@
+import express from 'express';
+import bcrypt from 'bcryptjs';
+
+import { addUserValidationRules, validateData } from '../validate.js';
+import { User } from '../models';
+
+const router = express.Router();
+
+const getUsers = async (request, response) => {
+  try {
+    const users = await User.findAll();
+    return response.status(200).send(users);
```

Comment:
Please leave a black line before `return` when there are other statements in that block of code. This improves readability.